wmfdata/spark.py

```diff
@@ -331,10 +343,47 @@ def run(commands, format="pandas", session_type="yarn-regular", extra_settings={
         elif format == "raw":
             overall_result = overall_result.collect()
 
-    # (re)start a timeout on SparkSessions in Yarn after the result is collected.
-    # A SparkSession used by this run function
-    # will timeout after 5 minutes, unless used again.
     if PREDEFINED_SPARK_SESSIONS[session_type]["master"] == "yarn":
-        start_session_timeout(spark_session, 3600)
+        start_session_timeout(spark_session)
 
     return overall_result
+
+def load_parquet(local_path, hive_db, hive_table=None, overwrite=False):
+    """
+    Loads a Parquet file into the Data Lake and registers it as a Hive table.
+
+    Arguments:
+    * `local_path`: the path (relative or absolute) to the file you want to load
+    * `hive_db`: the Hive database into which to place the loaded data. Must
+      already exist.
+    * `hive_table` (optional): the Hive table name to use for the loaded data.
+      If not given, the local file name is used (e.g. "experiment_data"
+      for "/dir/experiment_data.parquet").
+    * `overwrite` (optional): whether to overwrite the specified Hive table
```

Comment:
I was thinking if this is specified, the table name should be mandatory too, otherwise you might more easily accidentally overwrite an existing table.  It could still happen but maybe it makes you double check.